src/extension/edge/backgroundScriptsAPIBridge.js

```diff
@@ -0,0 +1,1050 @@
+try {
```

Comment:
can we have some sort of reference to where this file was obtained? same for the other bridge

Reply:
I'll add a comment pointing to the docs and the extension converter tool.